pkg/operator/clustermembercontroller2/clustermembercontroller.go

```diff
@@ -182,22 +196,22 @@ func (c *ClusterMemberController) reconcileMembers() error {
 		return nil
 	}
 
+	podFQDN, err := c.getValidPodFQDNToScale(unreadyPods)
```

Comment:
we will never set progressing condition if we kept erroring here. The time spent here is in order of a few seconds, while DNS entries are populated.